docs/management/admin/event-filters.asciidoc

```diff
@@ -4,6 +4,8 @@
 
 Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example, high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
```

Comment:
This isn't true and it feels like a non sequitur.  What is this sentence trying to convey?

Reply:
@ferullo I agree, that does seem out of place. I think it's related to a Tip a little further down the page, where we suggest filtering events to only display Endpoint events ("In the KQL search bar, enter the following query: `event.dataset : endpoint.events.network`.").

If that's not an accurate way to filter endpoint events, we should probably revise this TIp too. Is there a better query or method we could suggest?

Reply:
Ah, ok, that's an accurate tip on how to filter for Endpoint network events. All Endpoint _network_ events have that field/value.

Reply:
Thanks for clarifying, @ferullo. I just [pushed a change](https://github.com/elastic/security-docs/pull/2516/commits/d64b7fc997315911b3fafde715801ffad46a3d74) to remove the non sequitur from the first paragraph and revise the Tip below.